src/shared/hooks/use-context-init-message.ts

```diff
@@ -1,9 +1,12 @@
 import { createContext, useContext } from "react";
 import { IInitInteractive } from "@concord-consortium/lara-interactive-api";
 
-export const InitMessageContext = createContext<IInitInteractive | null>(null);
+export const InitMessageContext = createContext<IInitInteractive | null | undefined>(undefined);
 
 export const useContextInitMessage = () => {
   const initMessage = useContext(InitMessageContext);
+  if (initMessage === undefined) {
```

Comment:
Does this works because useInitMessage from the interactive client returns `null` when the init message isn't ready yet?  If that is true, then a comment to that effect would be useful here.

Reply:
Yeah, as I understand it, `useInitMessage` will return `null` when the init message isn't ready. See  https://github.com/concord-consortium/lara/blob/e05bc94cbe077833384463bc9a9b3a4b54e77deb/lara-typescript/src/interactive-api-client/hooks.ts#L105

I was going to ask what you thought of this change once the checks were done. Looks like they passed and the change seems to work as expected in my testing. Does it seem reasonable to you? I'll add a comment like you suggested if so.

Reply:
@emcelroy Yes this seems reasonable.